<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAlignRasterDialog</class>
 <widget class="QDialog" name="QgsAlignRasterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align Rasters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Raster layers to align</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="mBtnAdd">
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mBtnEdit">
       <property name="text">
        <string>/</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mBtnRemove">
       <property name="text">
        <string>-</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="mViewLayers">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Output size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Reference layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="mCboReferenceLayer"/>
     </item>
     <item row="1" column="2">
      <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true"/>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QgsDoubleSpinBox" name="mSpinCellSizeX">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsDoubleSpinBox" name="mSpinCellSizeY">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QgsDoubleSpinBox" name="mSpinGridOffsetX">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsDoubleSpinBox" name="mSpinGridOffsetY">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QCheckBox" name="mChkAddToCanvas">
       <property name="text">
        <string>Add aligned rasters to map canvas</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QgsExtentGroupBox" name="mClipExtentGroupBox">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="mEditOutputSize">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="mChkCustomCRS">
       <property name="text">
        <string>CRS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="mChkCustomCellSize">
       <property name="text">
        <string>Cell size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="mChkCustomGridOffset">
       <property name="text">
        <string>Grid offset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="mProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mBtnAdd</tabstop>
  <tabstop>mBtnEdit</tabstop>
  <tabstop>mBtnRemove</tabstop>
  <tabstop>mViewLayers</tabstop>
  <tabstop>mCboReferenceLayer</tabstop>
  <tabstop>mChkCustomCRS</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>mChkCustomCellSize</tabstop>
  <tabstop>mSpinCellSizeX</tabstop>
  <tabstop>mSpinCellSizeY</tabstop>
  <tabstop>mChkCustomGridOffset</tabstop>
  <tabstop>mSpinGridOffsetX</tabstop>
  <tabstop>mSpinGridOffsetY</tabstop>
  <tabstop>mClipExtentGroupBox</tabstop>
  <tabstop>mEditOutputSize</tabstop>
  <tabstop>mChkAddToCanvas</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsAlignRasterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
